new features

eur0pa opened the pull request:
a palla de foco ☄️

evilsocket commented:
zio finchè non fixi il nome del package e ci rimetti il mio non posso mergiare :)

evilsocket commented:
why are you removing the license?

eur0pa commented:
Sorry, I'm making a mess because of a GoLang bug; trying la Madonna and several saints to resolve the issue pertaining the dying filedescriptors and failing DNS queries, as reported in https://github.com/golang/go/issues/18588 and https://github.com/golang/go/issues/23866. Trying different approaches doesn't really work and another bug has been filed for, possibly, a resolution in 1.11. Between tests and approaches I removed a lot of stuff (test files, readme, license, moved dependencies around), with no ill-intent. The GoLang bug makes this PR (or any go-based "dirbuster" tool) useless and non-production ready, so I'm not giving too much care to everything surrounding the tool; I'll close the PR and reopen it with a proper license and an updated Readme when the time comes
